adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HttpAdapterMetrics.java

```diff
@@ -23,6 +23,7 @@
 public class HttpAdapterMetrics extends Metrics {
 
     private static final String SERVICE_PREFIX = "hono.http";
+    private static final String COMMANDS = ".commands";
```

Comment:
Since we have commands also in other components, I think this could go a level up. For the other messages we use a singular and also the endpoint names (telemetry, event). So maybe "control" and the constant from CommandConstants would be good? WDYT?

Reply:
Thank you for the comments.

1. I have pulled the COMMANDS to one level up to the Metrics.
2. Currently the word "messages" (.hono..messages*) is part of the metrics name to denote metrics corresponding to telemetry or event messages. To differentiate commands from them, I used the word "commands" as part of the metrics name (.hono.commands). If I change to "control" the series would look like ".hono.control". IMHO "commands" follows the similar naming convention as of "messages". Or do you think that commands should also be part of ".hono..messages*" with type as "control". I thought of this approach too. But the current approach is optimal also for the graph at the top in dashboard which shows no. of commands per tenant (aggregated across all adapter types). WDYT?